src/main/webapp/user.js

```diff
@@ -1,43 +1,52 @@
+import {
+  standardizeFetchErrors,
+  makeRelativeUrlAbsolute,
+} from './js/fetch_handler.js';
+
 /**
- * Get the user @param user from the user servlet. gets a user object from the
- * UserServlet
- * @param {string} user
+ * Get the user with @param userID from the user servlet. If no @param userID is
+ * supplied, function returns the currently signed in user.
+ * @param {string=} userID OPTIONAL
  * @return {dashboard.User}
  */
-function getUser(user) { //eslint-disable-line  
-  // TODO: switch to standard fetch error handler
-  return fetch('/user?githubID=' + user)
-      .then(errorHandling).then((response) => response.json())
+function getUser(userID = null) { //eslint-disable-line  
+  const fetchURL = '/user' + (userID ? '?userID=' + userID : '');
+  const fetchRequest = fetch(makeRelativeUrlAbsolute(fetchURL));
+
+  const errorFormattedFetchRequest = standardizeFetchErrors(
+      fetchRequest,
+      'Failed to communicate with the server. Please try again later.',
+      'An error occcured while retrieving this account.' +
+    ' Please try again later.');
+
+  return errorFormattedFetchRequest.then((response) => response.json())
       .then((user) => {
         return user;
-      })
-      .catch((error) => {
-        console.error(error);
-        return null;
       });
 }
 
 /**
  * Posts a user to the servlet
  * @param {URLSearchParams} userParams parameters with the users information
+ * @return {?Promise} Returns a prepared post user fetch request or null if the
+ * fetch request could not be prepared.
```

Comment:
This doesn't reflect the behavior of this function:
```suggestion
 * @return {Promise} Returns a prepared post user fetch request.
```